### PR TITLE
POC small change to add GET functionality to the rest api

### PR DESCRIPTION
**Pull request by mrengstrom**

### CODE REVIEW

File: app/api/rest_api.py [hunk 1]
```diff
@@ -68,9 +70,15 @@ async def landing(self, request):
     async def rest_core(self, request):
         try:
             access = dict(access=tuple(await self.auth_svc.get_permissions(request)))
-            data = dict(await request.json())
+            if request.method == 'GET':
+                data = dict(index='default', path=request.path[10:])
```
Comment (unkempthenry) [inline app/api/rest_api.py:74]: I think you can do `request.match_info['tail']` here instead of `request.path[10:]`

Comment (mrengstrom) [reply app/api/rest_api.py:74]: yea, that makes sense, let me try that out

**Comment by mrengstrom** [comment]
I'll just toss a comment up for hopefully seed things.  would we want to go the route of using the path or something like that, or would it preferred to pass in params, since our options are limited on GET requests?  I am leaning towards the latter in my head even though it is contrary to this PR.  Discuss!

**Comment by christophert** [comment]
I think we should migrate towards utilizing the path for objects. The URL should be used to identify unique objects and it's more intuitive from an API standpoint. Utilizing GET/POST/PATCH/DELETE verbs to act upon those objects makes more sense. URL parameters should only be used for additional criteria for locating collections of resources based on attributes.

**Comment by wbooth** [comment]
Great conversation starter. Closing since no recent activity

**Pull request closed without merging**